litmus/PTX/Memalloy/IRIW_gl_cta.litmus

```diff
@@ -0,0 +1,18 @@
+PTX IRIW_gl_cta
+"CoRR"
+"https://github.com/johnwickerson/memalloy/blob/master/ptx_testing/IRIW_gl_cta/IRIW_gl_cta.txt"
+{
+P0:r0=0;
+P1:r0=0;
+P1:r2=0;
+P2:r0=0;
+P3:r0=0;
+P3:r2=0;
+x=0;
+}
+ P0@cta 0,gpu 0           | P1@cta 1,gpu 0           | P2@cta 1,gpu 0           | P3@cta 1,gpu 0           ;
+ ld.weak r0, 1            | ld.weak r0, x            | ld.weak r0, 2            | ld.weak r0, x            ;
```

Comment:
`ld.weak r0, 2` should be `ld.weak r0, 1`

Reply:
fixed